external.bzl

```diff
@@ -0,0 +1,120 @@
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
```

Comment:
maybe worth a comment in here explaining what should go here vs directly in WORKSPACE